packages/python_solvespace/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: python_solvespace
+  version: 3.0.6
+source:
+  url: https://files.pythonhosted.org/packages/50/a6/4e01d2c90de41a431041f813aebada874d05d6d578ce25b2462872d2a3e6/python_solvespace-3.0.6.tar.gz
+  sha256: 951b4cf3858dc337bbfa9f6d9efe9fd6e8dfa7fed022556196add1ba5a56b045
+  extras:
+    - [Eigen/Cholesky, python_solvespace/include/Eigen/Cholesky]
```

Comment:
Please get rid of all of these extras. We will not vendor Eigen into Pyodide. I think you should be able to replace it with:
```yaml
build:
   script: |
      git clone https://gitlab.com/libeigen/eigen.git --depth 1 --branch 3.4
      ln -s eigen/Eigen python_solvespace/include/
```

Reply:
Thanks a lot, the build script works, I remove all extras in `meta.yaml` and add `test_python_solvespace.py` in https://github.com/pyodide/pyodide/pull/2287/commits/6177a5c96982f1b207f7b4b44b933a9b070b13bb. I had reported the missing file issue to KmolYuan,  he probably fix the issue in the next version of python-solvespace.